packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -72,19 +85,33 @@ This package contains documentation for rubygem-%{gem_name}.
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
+mkdir -p %{buildroot}%{_root_sbindir}
 cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
 rm %{buildroot}%{gem_instdir}/.babelrc
 
 %{foreman_bundlerd_file}
 %foreman_precompile_plugin -a -s
 
+ln -sv %{gem_instdir}/extra/cockpit/foreman-cockpit-session %{buildroot}%{_root_sbindir}/foreman-cockpit-session
+install -Dp -m0644 %{buildroot}%{gem_instdir}/extra/cockpit/foreman-cockpit.service %{buildroot}%{_unitdir}/foreman-cockpit.service
+install -Dp -m0644 %{buildroot}%{gem_instdir}/extra/cockpit/cockpit.conf.example %{buildroot}%{_root_sysconfdir}/foreman/cockpit/cockpit.conf
+install -Dp -m0644 %{buildroot}%{gem_instdir}/extra/cockpit/settings.yml.example %{buildroot}%{_root_sysconfdir}/foreman/cockpit/foreman-cockpit-session.yml
+
 %posttrans
 %{foreman_db_migrate}
 %{foreman_db_seed}
 %{foreman_apipie_cache}
 %{foreman_restart}
 exit 0
 
+%post cockpit
+%systemd_postun_with_restart foreman-cockpit.service
+%systemd_post foreman-cockpit.service
+
+%postun cockpit
+%systemd_postun_with_restart foreman-cockpit.service
+%systemd_post foreman-cockpit.service
```

Comment:
This also surprises me

Reply:
Ditto

Reply:
The `%systemd_postun_with_restart` should be sufficient here.

Reply:
Updated